xuluomin/client/src/client_node.cpp

```diff
@@ -0,0 +1,44 @@
+#include <client/Time.h>
+#include <client/show.h>
+#include <ros/time.h>
+#include <ros/ros.h>
+#include "std_msgs/Header.h"
+int main(int argc, char **argv) {
+    // ROS节点初始化
+    ros::init(argc, argv, "client_topic");
+    // 创建节点句柄
+    ros::NodeHandle n;
+
+    ros::ServiceClient cli = n.serviceClient<client::show>("show_name");
+    client::show srv;
+    srv.request.name ="show";
+
+    if(cli.call(srv)){
+        ROS_INFO("result: %s",  srv.response.res.c_str());
+        ROS_INFO("Succeed calling!");
+    } else {
+        ROS_ERROR("Failed to call");
+        return 1;
+    }
+
+
+    //ros::Publisher client_topic_pub = n.advertise<client::Time>(srv.response.res.c_str(), 10);
+
+    ros::Publisher client_topic_pub = n.advertise<std_msgs::Header>("head", 10);
+    // 循环的频率
+    ros::Rate loop_rate(1);
+
+    while (ros::ok()) {
+
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Redundant blank line at the start of a code block should be deleted.  [whitespace/blank_line] [2]